va_explorer/templates/va_data_management/index.html

```diff
@@ -30,5 +30,18 @@
     {% endfor %}
   </tbody>
 </table>
+{% if va_data.has_other_pages %}
```

Comment:
django-bootstrap has a template tag that will render a nice looking paginator in a bootstrap style. Maybe we could try that here? 

https://django-bootstrap4.readthedocs.io/en/latest/templatetags.html#bootstrap-pagination